doc/changelog.md

```diff
@@ -78,49 +90,63 @@
 
   ```pycon
   >>> circuit(0.5, 0.5)
-  >>> "Inside QNode: 0.5"
```

Comment:
Is the goal of the print statement to show recompilation? If so, should we do this:

```
from catalyst import qjit

dev = qml.device("lightning.qubit", wires=1)

@qjit(static_argnums=(1,))
@qml.qnode(dev)
def circuit(x, c):
    print("Inside QNode:", c)
    qml.RY(c, 0)
    qml.RX(x, 0)
    return qml.expval(qml.PauliZ(0))

print(circuit(0.5, 0.5))
print(circuit(0.2, 0.5))
print(circuit(0.2, 0.2))
```

```
Inside QNode: Traced<ShapedArray(float64[], weak_type=True)>with<DynamicJaxprTrace(level=2/1)>
0.7701511529340697
0.8600893382050472
Inside QNode: Traced<ShapedArray(float64[], weak_type=True)>with<DynamicJaxprTrace(level=2/1)>
0.9605304970014428
```

Reply:
Where

```
print(circuit(0.5, 0.5)) # compilation step
print(circuit(0.2, 0.5)) # no recomp needed
print(circuit(0.2, 0.2)) # recomp needed because static arg val changes
```

Reply:
Nope, the goal of the print statement is to show that `c` is statuc (it is not a `Traced<ShapedArray>`). The reason it is showing up like that on yours is that `dev1` on test-pypi was uploaded before this bugfix!

I've edited this example to make it a bit clearer